importer/Import/Repodata.hs

```diff
@@ -75,6 +75,14 @@ extractType doc dataType = let
 fetchAndParse :: (MonadThrow m, MonadIO m, MonadBaseControl IO m) => URI -> m Document
 fetchAndParse uri = runConduitRes $ getFromURI uri .| ungzipIfCompressed .| sinkDoc def
 
+addSlash :: URI -> URI
+addSlash u = do
```

Comment:
You shouldn't need a "do" here, since there's no monad involved.  In that case this just becomes:
```
addSlash u = let
    path = uriPath u
 in
    if ...
```